contributing/giving-constructive-criticism.rst

```diff
@@ -0,0 +1,155 @@
+Giving constructive criticism
+=============================
+
+:doc:`Reviewing issues and pull requests </contributing/community/reviews>`
+is a great way to get started with contributing to the Symfony community.
+Anyone can do it! But before you give a comment, take a step back and think,
+is what you are about to say actually what you intend?
+
+Communicating over the Internet with nothing but text can pose a
+big challenge, especially if you remember that the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+ideas and opinions.
+
+Not everyone speaks English or is able to use a keyboard, some might
+have Dyslexia or similar conditions that affect their writing.
+
+Not to mention that some might have a bad experience from previous
+contributions (to other projects).
+
+You're not alone in this. This guide will try to help you write
+constructive, respectful and helpful reviews and replies.
+
+.. tip::
+
+    This guide is not about lecturing you to "conform" or give-up
+    your ideas and opinions but helping you to better communicate,
+    prevent possible confusion, and keeping the Symfony community a
+    welcoming place for everyone. **You are free to disagree with
+    someone's opinions, just don't be disrespectful.**
+
+First of, accept that many programming decisions are opinions.
+Discuss trade offs, which you prefer, and reach a resolution quickly.
+It's not about being right or wrong, but using what works.
+
+Tone of voice
+-------------
+
+We don't expect you to be completely formal, or to even write error-free
+English. Just remember one thing: Don't swear, and be respectful to others.
+
+Don't reply in anger or with an aggressive tone. You're angry, we understand
+that, but swearing/cursing and name calling doesn't really encourage anyone to
+help you. Take a deep breath, count to 10 and try to *clearly* explain what problems
+you encounter.
+
+Gender pronouns
+---------------
+
+While not "formally" required it's better to use gender-neutral pronouns.
+Unless someone "indicated" their pronouns, use "they", "them" instead of
+"he", "she", "his", "hers", "his/hers", "he/she", etc.
+
+And instead of saying "guys" it's better to use "folks", or Symfonions
+if you like that.
+
+Giving positive feedback
+------------------------
+
+While reviewing issues and pull request you may run into some suggestions
+(including patches) that don't reflect your ideas, are not good, or downright wrong.
+
+Now, instead of writing a lengthy comment why you think they are wrong,
```

Comment:
I'm not sure I understand this paragraph well. Is it saying that if a pull request contains lots of changes you shouldn't say that it's wrong ... even when it's wrong ("wrong" according to Symfony philosophy, established practices, etc.)?

By the way, somewhere we should remind people to ALWAYS create an issue first to debate about their idea and not submit a pull request directly.